src/test/java/io/github/christophermanahan/carnitas/RequestParserTest.java

```diff
@@ -72,7 +72,7 @@ void itIsEmptyIfReaderIsEmpty() {
         private final Iterator<String> request;
 
         RequestReader(String request) {
-            this.request = List.of(request.split(HTTPResponse.CRLF, -1)).iterator();
+            this.request = List.of(request.split(Serializer.CRLF, -1)).iterator();
```

Comment:
What is `-1` mean? Perhaps we should extract it into a temporary variable to reveal the meaning?